backend/src/v4/handler/db.js

```diff
@@ -105,11 +105,13 @@
 	};
 
 	Handler.dropCollection = function (database, collection) {
```

Comment:
I think removeModelCollections method in  'v5/utils/helper/models' is calling this method without passing the name but the whole collection obj